searches.view.lkml

```diff
@@ -439,91 +414,166 @@ view: searches {
+  dimension: search_field {
+    type: string
+    # sql: decode(split_part(${page_url},'/search/',2),'%20', ' ');;
+    sql: REPLACE(split_part(${page_url},'/search/',2), '%20', ' ')
+      ;;
+    hidden: yes
+  }
+
   # Search results
 
+  # terms: string
+  #
+  # Table reference - VARCHAR(2048) ENCODE RAW
   # terms returns a string array of search terms
-  # which can have a size > 1
+  # The array can have a size greater than 1 in multi-dimension searches
+  # TODO: GDXDSD-1326
   dimension: terms {
     type: string
     sql: ${TABLE}.terms ;;
     group_label: "Results"
   }
 
-  # readable_terms removes wrapping [" "] from the terms dimension,
-  # array and replaces + signs with ' ', rendering a readable string
-  # in the case of the term string array containing a single element
+  # readable_terms: string
+  #
+  # removes wrapping [" "] from the terms dimension, array and
+  # replaces + signs with ' ', rendering a readable string in
+  # the case of the term string array containing a single element
+  # TODO: GDXDSD-1326 handle multiple-term searches past the first index, and empty searches.
   dimension: readable_terms {
     type:  string
     sql:  replace(substring(${terms}, 3, length(${terms})-4),'+',' ') ;;
     group_label: "Results"
   }
 
+  # filters: string
+  # Table reference - VARCHAR(2048) ENCODE RAW
+  # e.g.: '{'category': 'books', 'sub-category': 'non-fiction'}'
   dimension: filters {
+    description: "The search filters selected."
     type: string
     sql: ${TABLE}.filters ;;
     group_label: "Results"
   }
 
+  # total_results: number
+  # Table reference - INT ENCODE RAW
   dimension: total_results {
+    description: "The number of search results found."
     type: number
     sql: ${TABLE}.total_results ;;
     group_label: "Results"
   }
 
+  # page_results: number
+  # Table reference - INT ENCODE RUNLENGTH
   dimension: page_results {
+    description: "The number of results displayed on the first page"
     type: number
     sql: ${TABLE}.page_results ;;
     group_label: "Results"
   }
 
-  # Device
-
-  dimension: device_type {
-    type: string
-    sql: ${TABLE}.dvce_type ;;
-    group_label: "Device"
-  }
-
-
-  dimension: device_ismobile {
-    type: yesno
-    sql: ${TABLE}.dvce_ismobile ;;
-    group_label: "Device"
-  }
-
   # MEASURES
+  # duplicates throughout this section
 
   measure: row_count {
     type: count
     group_label: "Counts"
   }
 
+  # search_count
+  # a distinct count of the unique searches returned
+  # Unique ID for each click (equal to event_id in atomic.events)
   measure: search_count {
+    description: "A distinct count of the unique searches returned in this Explore."
     type: count_distinct
     sql: ${search_id} ;;
     group_label: "Counts"
   }
 
+  # max_search_index: number
+  # Table reference - INT8 ENCODE ZSTD
+  # returns the max value of search session indexes
   measure: max_search_index {
```

Comment:
Should the index be hidden?

Reply:
In the same vein as on your `max_click_index` question; I disagree that `max_search_index` since it is a well described calculation on `search_in_session_index`; which itself _is_ hidden.